pocketoptionapi_async/client.py

```
@@ -364,146 +418,168 @@ async def place_order(self, asset: str, amount: float, direction: OrderDirection
                     if self.auto_reconnect:
                         reconnected = await self._attempt_reconnection()
                         if reconnected:
-                            logger.info(f"✅ Reconnected, retrying candle request for {asset}")
+                            logger.info(
+                                f" Reconnected, retrying candle request for {asset}"
+                            )
                             continue
-                    
+
                 logger.error(f"Failed to get candles for {asset}: {e}")
                 raise PocketOptionError(f"Failed to get candles: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)